bin/build.sh

```diff
@@ -1,43 +1,59 @@
 #!/bin/bash
 
+# Makes it so that all commands are displayed as they get executed
 set -x
 
 wca_url="https://worldcubeassociation.org/"
+stylesheet=$(realpath "./assets/style.css")
 
 # Find formal and legal Markdown files and build PDFs out of them.
 find 'documents' -name '*.md' | while read file; do
   echo "Processing $file, vanilla style"
 
   html_name="${file%.md}.html"
   pdf_name="${file%.md}.pdf"
+  
+  file_headline=$(head -n 1 "$file")
+  document_title=$(echo "$file_headline" | sed -E "s/#+\s*//")
+  
+  # Copy stylesheet to be used by the temporary HTML files
+  cp "$stylesheet" "$(realpath $(dirname "$file"))"
 
   # deliberately NOT using the pipe | operator here because wkhtml tries to resolve image links
   # based on the location of its input file. If piping is used, wkhtml assumes /tmp as location
   # and links like "./dummy-image.png" become "/tmp/dummy-image.png" instead of "edudoc/$project/dummy-image.png"
-  sed -r "s#wca\{([^}]*)\}#$wca_url\1#g" "$file" | # Replace wca{...} with absolute WCA URL.
-  pandoc -o "$html_name" # Markdown -> HTML
-  wkhtmltopdf --encoding 'utf-8' --user-style-sheet 'assets/style.css' -T 15mm -B 15mm -R 15mm -L 15mm --quiet "$html_name" "$pdf_name" # HTML -> PDF
```

Comment:
What was wrong with this; it seemed to be working well so what is the motivation  for the changes?

Reply:
As I mentioned in my PR, wkhtmltopdf doesn't apply styling properly, as mentioned [here](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2469) as well. This makes the tables not render as expected. Pandoc doesn't have the same issue, so I applied styling in there and it worked flawlessly and even seems to have fixed a couple other minor quirks.

Reply:
Thanks for the clarification!